tests/Tests/500__jupyterhub/test-jupyterlab-flask-notebook.robot

```diff
@@ -0,0 +1,85 @@
+*** Settings ***
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Library          DebugLibrary
+Suite Setup      Begin Web Test
+Suite Teardown   End Web Test
+
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Library          DebugLibrary
+Library          JupyterLibrary
+
+
+*** Variables ***
+
+
+*** Test Cases ***
+Open RHODS Dashboard
+  [Tags]  Sanity
```

Comment:
Is this test case supposed to be in Sanity? @vasukulkarni

Reply:
@lugi0 I changed the tags all to "Tier2"